### PR TITLE
RAD-3446 Fix MailingListServiceTest when DST switchover

### PR DESCRIPTION
**Pull request by MertCingoz**

### CODE REVIEW

File: Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java
Comment (benji811124) [inline Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java:1]: The change would be specific for this year. Am I correct?

Comment (MertCingoz) [reply Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java:1]: It is still using current week (dynamic) as before, I added extra 2 dates from this year to show test is expecting correct IntervalId

**Review by benji811124 — APPROVED** [review]

**Comment by terrypacker** [comment]
@MertCingoz some description of what you did in this PR would be good.  It looks like there was a bug in the test and not the code?

File: Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
  */
 package com.infiniteautomation.mango.spring.service;
 
+import static org.junit.Assert.*;
```
Comment (terrypacker) [inline Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java:6]: We shouldn't be adding wildcard imports.  This should be configured in your IDE. Please check the setup doc and fix.

**Review by terrypacker — CHANGES_REQUESTED** [review]

**Comment by MertCingoz** [comment]
> @MertCingoz some description of what you did in this PR would be good. It looks like there was a bug in the test and not the code?

This test predicts that intervalIds will increase by 1 at a time, but it is not valid for daylight saving times.
I don't see other problems in code. IntervalId and WeeklySchedule looks not related with timezone at all. [InactiveIntervalConversionTest](https://github.com/MangoAutomation/ma-modules-public/blob/main/Mango%20API/src-test/com/infiniteautomation/mango/rest/latest/model/mailingList/InactiveIntervalConversionTest.java)

File: Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java (hunk 1 above)
Comment (MertCingoz) [reply Core/src-test/com/infiniteautomation/mango/spring/service/MailingListServiceTest.java]: I missed the static import setting, thanks.

**Review by terrypacker — APPROVED** [review]

**Pull request merged**